cluster-enroll-cobbler.sh

```diff
@@ -49,7 +49,7 @@ if [[ -f cluster.txt ]]; then
 		MATCH="$HOSTNAME"
 		ACTIONSTRING="adding $HOSTNAME.$DOMAIN ($IPADDR,$MACADDR) to cobbler with profile ${PROFILE}..."
 		echo $ACTIONSTRING | tee -a $TRANSCRIPT
-		sudo cobbler system add --name=$HOSTNAME --hostname=$HOSTNAME.$DOMAIN --profile=$PROFILE --ip-address=$IPADDR --mac=$MACADDR
+		sudo cobbler system add --name=$HOSTNAME --hostname=$HOSTNAME.$DOMAIN --profile=$PROFILE --ip-address=$IPADDR --interface=eth0 --mac=$MACADDR
```

Comment:
This is the DHCP interface used by clients after the OS comes up.   It is a mandatory option in Cobbler 2.6.  It's hardcoded to `eth0` because that was the de facto behavior in versions of cobbler prior to 2.6.